### PR TITLE
v9.4.0

### PR DESCRIPTION
**Pull request by adamraine**

### CODE REVIEW

File: changelog.md [hunk 1]
```diff
@@ -1,3 +1,49 @@
+<a name="9.4.0"></a>
+# 9.4.0 (2022-02-16)
+[Full Changelog](https://github.com/compare/v9.3.1...v9.4.0)
+
+We expect this release to ship in the DevTools of [Chrome 100](https://chromiumdash.appspot.com/schedule), and to PageSpeed Insights within 2 weeks.
+
+## Notable Changes
+
+* fr: user triggered navigations ([#13496](https://github.com/GoogleChrome/lighthouse/pull/13496))
```
Comment (connorjclark) [inline changelog.md:9]: - if this is notable, we should say more about it
- remove "fr: "

File: changelog.md [hunk 2]
```diff
@@ -1,3 +1,49 @@
...
+
+## Core
+
+* a11y: change link in category description to web.dev ([#13638](https://github.com/GoogleChrome/lighthouse/pull/13638))
+* full-page-screenshot: wait for doubleraf, network quiet ([#13663](https://github.com/GoogleChrome/lighthouse/pull/13663))
+* hreflang: remove eval, import axe valid-langs.js directly ([#13385](https://github.com/GoogleChrome/lighthouse/pull/13385))
+* inputs: fix typo in artifact ([#13671](https://github.com/GoogleChrome/lighthouse/pull/13671))
+* inputs: refactor form-elements gatherer ([#13662](https://github.com/GoogleChrome/lighthouse/pull/13662))
```
Comment (connorjclark) [inline changelog.md:17]: combine

File: changelog.md [hunk 3]
```diff
@@ -1,3 +1,49 @@
...
+* page-functions: set style in getOuterHTMLSnippet without violating CSP ([#13636](https://github.com/GoogleChrome/lighthouse/pull/13636))
```
Comment (connorjclark) [inline changelog.md:18]: could consider this notable (fixes a somewhat high viz bug)

**Review by connorjclark — APPROVED** [review]

File: changelog.md (hunk 3 above)
Comment (adamraine) [reply changelog.md:17]: Eh I think it's fine here

**Pull request merged**